module-2/utils/index.ts

```diff
@@ -0,0 +1,14 @@
+import User from '../models/User';
+
+export const filterNotDeleted = (user: User): boolean => !user.isDeleted;
```

Comment:
A similar function might be used in "get User by Id" handler.

Reply:
Used a newly created `isMissing` check in "get User by Id"